src/porepy/examples/tracer_flow.py

```diff
@@ -0,0 +1,275 @@
+"""Module containing a simple tracer flow setup, modelled as a single phase, 2-component
```

Comment:
I understand. My motivation was the following:

- The model in the tutorial accompanied by more detailed explanations about why individual steps are required. Using a "finished" model from `src` would make that tutorial messy.
- The tracer flow in `porepy.examples` is there to be used, by students or anyone interested.
- The model in `tests` is there because I was a bit lazy (in all honesty), I was expecting MRO issues because the test case has a different grid, IC and BC. I think I can eliminate that set-up with some extra effort and proper mixing.

Reply:
Solution: This code stays, the tutorial uses inspect.

Reply:
Tutorial uses the `examples.tracer_flow` and uses inspect.
The tests also use the `examples.tracer_flow.TracerFlowSetup` now.
No more code duplication in tutorial or tests. Basis is example model in the core.